Week1/homework/6-log-an-an-array-of-animals.js

```diff
@@ -0,0 +1,20 @@
+//Declare variable and assign to it an empty array. Make sure that the name of the variable indicates it contains more than 1 item. For example items instead of item.
+let friends = [];
+
+//Write a console.log statement that explains in words what you think the value of the array is.
+console.log("the value of the array is an empty array")
+
+//Write a console.log statement that logs the array.
+console.log(friends);
+
+//Create a new variable with an array that has 3 of your favorite animals, each in a different string. Make sure the name of the variables says something about what the variable contains.
+let favAnimals = ["cats", "birds", "sheeps"];
+
+//Write a console.log statement that logs the second array.
+console.log(favAnimals);
+
+//Add a statement that adds another string ("Piglet)" to the array of animals.
+favAnimals[3] = "Piglet";
```

Comment:
A better way of achieving this effect would be to use the `push` method. For instance: `favAnimals.push("Piglet");`